dpos/state/state.go

```diff
@@ -22,22 +22,22 @@ const (
 	// confirmations yet.
 	Pending ProducerState = iota
 
-	// Activate indicates the producer is registered and confirmed by more than
+	// Active indicates the producer is registered and confirmed by more than
 	// 6 blocks.
-	Activate
+	Active
 
-	// Inactivate indicates the producer has been inactive for a period which shall
-	// be punished and will be activate later
-	Inactivate
+	// Inactive indicates the producer has been inactivated for a period which shall
+	// be punished and will be activated later
+	Inactive
 
 	// Canceled indicates the producer was canceled.
 	Canceled
 
-	// FoundBad indicates the producer was found doing bad.
-	FoundBad
+	// Illegal indicates the producer was found to break the consensus.
+	Illegal
 
-	// ReturnedDeposit indicates the producer has canceled and returned deposit
-	ReturnedDeposit
+	// Returned indicates the producer has canceled and returned deposit
```

Comment:
Returned indicates the producer has canceled and deposit returned.

Reply:
fixed